code/game/objects/structures/girders.dm

```diff
@@ -59,14 +59,14 @@
 		if(istype(W, /obj/item/stack/rods))
 			var/obj/item/stack/rods/S = W
 			if(state == GIRDER_DISPLACED)
-				if(S.get_amount() < 2)
+				if(S.get_amount() < 5)
```

Comment:
Undocumented change? What is this for?

If this is for the duplicating rods change, this is an odd way to fix it, and feels like it should be a separate PR entirely.

Reply:
#35065 didnt do what it was meant to do. I was touching relevant code so i just slapped it in there.

Reply:
Basing this off of #35065. It appears that it was simply deprecated so i just slapped it on there.

Reply:
Oh, I see, this is for RODS, not metal. I don't care then